ArchiSteamFarm/IPC/Startup.cs

```diff
@@ -211,17 +213,22 @@ internal sealed class Startup {
 
 					knownNetworks.Add(new IPNetwork(ipAddress, prefixLength));
 				}
+			} else {
+				// Use private address space networks by default, https://datatracker.ietf.org/doc/html/rfc1918#section-3
+				knownNetworks.Add(new IPNetwork(IPAddress.Parse("10.0.0.0"), 8));
+				knownNetworks.Add(new IPNetwork(IPAddress.Parse("172.16.0.0"), 12));
+				knownNetworks.Add(new IPNetwork(IPAddress.Parse("192.168.0.0"), 16));
 			}
 
+			KnownNetworks = knownNetworks.ToImmutableHashSet();
```

Comment:
Can we extract the whole creation of `knownNetworks` into a function?

Reply:
I don't believe it's justified. We don't need 40 10-lines functions, this block is quite simple to understand and it's part of the configuration.